app/src/main/res/layout/content_drawer.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
this XML doesn't require constraints since it only has one child. You can use a FrameLayout for better performance

Reply:
done